sample-queries/sample-queries.json

```diff
@@ -204,7 +204,69 @@
         "requestUrl": "/v1.0/groups",
         "docLink": "https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/resources/group",
         "skipTest": false
+    },    
+    {
+        "id": "6433ec9c-1ad5-4245-a5f2-8136c79856a9",
+        "category": "Groups",
+        "method": "GET",
+        "humanName": "list all groups in my organization",
+        "requestUrl": "/v1.0/groups",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/group-list?view=graph-rest-1.0&tabs=http",
+        "tip": "Please provide us with feedback on the groups API here: https://aka.ms/GroupsAPIFeedback",
+        "skipTest": false
+    },
+    {
+        "id": "ed4996d8-318a-45f2-a036-0abb52640d19",
+        "category": "Groups",
+        "method": "GET",
+        "humanName": "get properties and relationships of group",
+        "requestUrl": "/v1.0/groups/{id}",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/group-get?view=graph-rest-1.0&tabs=http",
+        "tip": "Please provide us with feedback on the groups API here: https://aka.ms/GroupsAPIFeedback",
+        "skipTest": false
+    },
+    {
+        "id": "b15626bb-d583-4505-a7ee-85f48f1710f4",
+        "category": "Groups",
+        "method": "POST",
+        "humanName": "create a new group",
+        "requestUrl": "/v1.0/groups",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/group-post-groups?view=graph-rest-1.0&tabs=http",
+        "tip": "Please provide us with feedback on the groups API here: https://aka.ms/GroupsAPIFeedback",
+        "postBody": "{\r\n  \"displayName\": \"Library Assist\",\r\n  \"mailEnabled\": true\",\r\n  \"mailNickname\": \"library\",\r\n  \"securityEnabled\": true\"\r\n}",
+        "skipTest": false
+    },    
+    {
+        "id": "1f7e23f9-55e2-4c4f-8506-c6bdb5a502c1",
+        "category": "Groups",
+        "method": "POST",
+        "humanName": "add member to group",
+        "requestUrl": "/v1.0/groups/{group-id}/members/$ref",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/group-post-members?view=graph-rest-1.0&tabs=http",
+        "tip": "Please provide us with feedback on the groups API here: https://aka.ms/GroupsAPIFeedback",
+        "postBody": "{\r\n  \"@odata.id\":\"https:\/\/graph.microsoft.com\/v1.0\/directoryObjects\/{id}\"\r\n}",
+        "skipTest": false
+    },    
+    {
+        "id": "c165c350-26ab-41bb-97d6-0e8b3fec051e",
+        "category": "Groups",
+        "method": "DELETE",
+        "humanName": "remove member from group",
+        "requestUrl": "/v1.0/groups/{id}/members/{id}/$ref",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/group-delete-members?view=graph-rest-1.0&tabs=http",
+        "tip": "Please provide us with feedback on the groups API here: https://aka.ms/GroupsAPIFeedback",
```

Comment:
Might be great to also add an additional tip message letting users know how to get the {id} described.

Also the {id should be descriptive e.g. {message-id}

Please do the same for all queries that need an {id}

Reply:
Thanks @bettirosengugi - I've updated the {id} to make it more descriptive,

Reply:
This is great @sanana30 please add an extra line on the tip message letting users know which request url they need to run to get the id e.g "To get group-id run http://graph.microsoft.com//v1.0/groups/ "

Reply:
@bettirosengugi - Would this need to be added to the same tip message that has our survey link? Or would I add a new tip message?

Reply:
@sanana30 same tip message. Just add a period at the end of the feedback string then write the other.